thoth/package_extract/image.py

```diff
@@ -546,6 +546,47 @@ def get_image_size(path: str) -> int:
     return total_size
 
 
+def _get_cuda_version(path: str) -> dict:
+    """Get the cuda version."""
+    res = dict()
+    # gathering version from version.txt file
+    version_path = path + "/usr/local/cuda/version.txt"
+    if os.path.isfile(version_path):
+        with open(version_path, "r") as f:
+            output = f.read()
+            lines = output.split()
+            if len(lines) != 3:
+                _LOGGER.info(
+                    "Unable to detect CUDA version from usr/local/cuda/version.txt output: %r",
+                    output,
+                )
+            else:
+                res["/usr/local/cuda/version.txt"] = lines[-1]
+
+    # gathering version from nvcc command
+    nvcc_path = glob.glob(path + '/usr/local/cuda-*/bin/nvcc')
+    if len(nvcc_path) > 0:
+        st = os.stat(nvcc_path[0])
+        os.chmod(nvcc_path[0], st.st_mode | stat.S_IEXEC)
+        result = run_command("{} --version".format(nvcc_path[0]), raise_on_error=False)
+        if result.return_code != 0:
+            _LOGGER.info(
+                "Unable to detect CUDA version - nvcc returned non-zero version: %s",
+                result.to_dict(),
+            )
+        else:
+            lines = result.stdout.splitlines()
```

Comment:
`lines` would be equal to something like `['nvcc: NVIDIA (R) Cuda compiler driver', 'Copyright (c) 2005-2019 NVIDIA Corporation', 'Built on Sun_Jul_28_19:07:16_PDT_2019', 'Cuda compilation tools, release 10.1, V10.1.243']`